src/jit/gentree.h

```diff
@@ -3284,6 +3284,9 @@ struct GenTreeCall final : public GenTree
 #define GTF_CALL_M_TAILCALL_VIA_HELPER 0x0200 // GT_CALL -- call is a tail call dispatched via tail call JIT helper.
 #endif                                        // !LEGACY_BACKEND
 
+#define GTF_CALL_M_FAT_POINTER_CHECK 0x0200 // GT_CALL -- it is calli that needs to be tranformed
+                                            // to check CoreRT fat pointer case.
```

Comment:
Please add a couple of lines explaining what is CoreRT fat pointer calli here to serve as some context/documentation.